### ISSUE TITLE
 How to solve this error?

### ISSUE THREAD

**Issue by BennyAlex** [rollup/rollup-plugin-babel#156]
I get this error:  (babel plugin) TypeError: Property declaration of ExportDefaultDeclaration expected node to be of a type ["FunctionDeclaration","ClassDeclaration","Expression"] but instead got "ExpressionStatement" src\colorIsLight.js

And my file colorIsLight:

```
function toRgb(color) {
  if (color[0] === '#') {
    color = color.substr(1)
  }
  if (color.length !== 6) return console.warn('color must have a length of 6 hex numbers')
  let r, g, b
  let rgb = parseInt(color, 16)   // convert rrggbb to decimal
  r = (rgb >> 16) & 0xff  // extract red
  g = (rgb >> 8) & 0xff  // extract green
  b = (rgb >> 0) & 0xff  // extract blue
  return {r: r, g: g, b: b}
}

export default function (color, differenceRange) {
  let rgb = toRgb(color) // https://www.w3.org/TR/AERT#color-contrast
  return ((rgb.r * 320 + rgb.g * 590 + rgb.b * 110) / 1000) > (differenceRange || 125) // suggested by w3
}

```

**Comment by nicolo-ribaudo** [comment]
As a workaround you can try

```js
function colorIsLight(color, differenceRange) {
let rgb = toRgb(color) // https://www.w3.org/TR/AERT#color-contrast
return ((rgb.r * 320 + rgb.g * 590 + rgb.b * 110) / 1000) > (differenceRange || 125) // suggested by w3
}

export default colorIsLight;
// Or, if even that doesn't work,
export { colorIsLight as default };
```

**Comment by BennyAlex** [comment]
@nicolo-ribaudo both not working :/

**Comment by gauntface** [comment]
I am hitting this same issue.

**Comment by arlac77** [comment]
starting with rollup-plugin-babel@3.0 i am getting the same error (for lots of different modules)

**Comment by nicolo-ribaudo** [comment]
If anyone claims this issue before, I can start looking for a solution the 13th

**Comment by nicolo-ribaudo** [comment]
@BennyAlex @gauntface @arlac77 @Riim @petamoriken Are you using babel 6? (`npm show babel-core` or `yarn show babel-core`)

**Comment by arlac77** [comment]
@nicolo-ribaudo am using babel-core@6.25.0

**Comment by Riim** [comment]
version: '6.25.0'

**Comment by nicolo-ribaudo** [comment]
Working workaround: use Babel 7.
You need to  `npm install --save-dev babel-core@alpha.19` and upgrade the various plugins to the same version. If it still doesn't work, try to delete the `node_modules` folder and run `npm install` (idk why, but I had some issue with npm package hoisting while testing this bug)

**Comment by BennyAlex** [comment]
@nicolo-ribaudo npm install --save-dev babel-core@alpha.19 does not work.
I get no matching version found

**Comment by nicolo-ribaudo** [comment]
Sorry, babel-core@7.0.0-alpha.19.

Anyway, I opened a PR to fix this bug with Babel 6 (#158)

**Comment by BennyAlex** [comment]
@nicolo-ribaudo unfortunatelly the PR isnt merged yet and I need it now. But with babel 7 I get unexpected token in .vue files, but I excluded .vue files in rollup config :/
